controllers/index.js

```diff
@@ -50,3 +51,18 @@ exports.postCommentByReviewId = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.patchReviewByReviewId = (req, res, next) => {
+  const {
+    params: { review_id },
+    body: { inc_value },
+  } = req;
+  Promise.all([
```

Comment:
I know why you've done this hear but you could use the row count in the model rather than making 2 requests to the database and reject in the model, I think it would be cleaner